src/features/Endpoint/Endpoint.tsx

```diff
@@ -36,6 +36,10 @@ const EndPoint = () => {
     window.location.reload();
   };
 
+  const server_url = localStorage.getItem('config.server_url');
+  const app_id = localStorage.getItem('config.app_id');
+  const current_url = `wss://${server_url}/websockets/v3?app_id=${app_id}&l=EN&brand=deriv`;
```

Comment:
```suggestion
  const server_url = localStorage.getItem('config.server_url') ?? default_endpoint.server_url;
  const app_id = localStorage.getItem('config.app_id') ?? default_endpoint.app_id;
  const current_url = `wss://${server_url}/websockets/v3?app_id=${app_id}&l=EN&brand=deriv`;
```
on first interaction with the website, since we have no values in localstorage. `server_url` and `app_id` will be `undefined`.

Reply:
Alright.Thanks